Dalamud/Dalamud.cs

```diff
@@ -187,27 +186,6 @@ public void WaitForUnload()
         this.unloadSignal.WaitOne();
     }
 
-    /// <summary>
-    /// Dispose subsystems related to plugin handling.
-    /// </summary>
-    public void DisposePlugins()
-    {
-        // this must be done before unloading interface manager, in order to do rebuild
```

Comment:
Are these dependencies still upheld? Who is responsible for it now?

Reply:
This function hasn't been called for a while, so it's probably safe to remove this at this point.
* IME: WndProc hook manager now exists separately, and it'll deal with clwaning up WndProc hooks.
* IM: On its own Dispose, it'll clean its rendering hooks from the Framework thread if it's active, so there's no chance PresentDetour getting called after that. If the game is exiting, Present calls are just over, so it doesn't matter anyway.

Reply:
Though, it probably is for the better to move that comment to IM.DisposeService. Will do that later.